coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -197,3 +197,22 @@ def from_metadata(cls, doccomment, docstyle_definition,
 
         return DocumentationComment(assembled_doc, docstyle_definition, indent,
                                     marker, range)
+
+    def assemble(self):
+        """
+        Assembles parsed documentation to the original documentation.
+
+        This function assembles the whole documentation comment, with the
+        given markers and indentation.
+        """
+        lines = self.documentation.splitlines(keepends=True)
+        assembled = self.indent + self.marker[0]
+        if len(lines) == 1:
+            assembled += lines[0]
+        else:
+            assembled += ''.join('\n' if line == '\n' and not self.marker[1]
+                                 else self.indent + self.marker[1] + line
+                                 for line in lines)
```

Comment:
It definitely doesn't make it easy. :3
The thing is, it is for python stuff.

I think a custom solution would work better to work across all languages and styles. :+1: